### PR TITLE
Simulateinstall

### PR DESCRIPTION
**Pull request by EvangelosG**
First shot at simulating installs w/o setDebug.

### CODE REVIEW

File: Branch-SDK/src/io/branch/referral/SystemObserver.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import android.provider.Settings.Secure;
 import android.text.TextUtils;
 import android.util.DisplayMetrics;
+import android.util.Log;
```
Comment (sojanpr) [inline Branch-SDK/src/io/branch/referral/SystemObserver.java]: Unnecessary import. I guess we don't need this in this class

**Comment by sojanpr** [comment]
Can we remove the gradle files added here?

**Comment by EvangelosG** [comment]
removed Log import, reverted gradle changes.

**Comment by sojanpr** [comment]
LGTM 👍

**Pull request merged**